site/content/docs/dashboards/authentication.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Private Dashboards
+weight: 48
+menu:
+  resources:
+    parent: "Dashboards"
+---
+
+## Authentication
+
+By default Checkly dashboards are public and visible to anyone who knows the URL. You can now set it to "Private", however, and protect it with an auto-generated password.
```

Comment:
I'd change the second line to something like: 'You can set a dashboard to private, limiting access to users with a password, or Checkly account,' or similar.
The intent is to make the documentation more 'neutral' from a freshness perspective, 'Now you can' implies to me that the feature is new, which it'll only be for a short while. Let's try to make the wording relevant for as long as we can.

Reply:
Yeah good call, updated :+1: